_inc/lib/debugger/class-jetpack-debug-data.php

```diff
@@ -0,0 +1,345 @@
+	 *
+	 * @return string The plan slug.
+	 */
+	public static function what_jetpack_plan() {
+		$plan = Jetpack_Plan::get();
+		return ! empty( $plan['class'] ) ? $plan['class'] : 'undefined';
+	}
+
+	/**
+	 * Convert seconds to human readable time.
+	 *
+	 * A dedication function instead of using Core functionality to allow for output in seconds.
+	 *
+	 * @since 7.3.0
+	 *
+	 * @param int $seconds Number of seconds to convert to human time.
+	 *
+	 * @return string Human readable time.
+	 */
+	public static function seconds_to_time( $seconds ) {
+		$seconds = intval( $seconds );
+		$units   = array(
+			'week'   => WEEK_IN_SECONDS,
+			'day'    => DAY_IN_SECONDS,
+			'hour'   => HOUR_IN_SECONDS,
+			'minute' => MINUTE_IN_SECONDS,
+			'second' => 1,
+		);
+		// specifically handle zero.
+		if ( 0 === $seconds ) {
+			return '0 seconds';
+		}
+		$human_readable = '';
+		foreach ( $units as $name => $divisor ) {
+			$quot = intval( $seconds / $divisor );
+			if ( $quot ) {
+				$human_readable .= "$quot $name";
+				$human_readable .= ( abs( $quot ) > 1 ? 's' : '' ) . ', ';
+				$seconds        -= $quot * $divisor;
+			}
+		}
+		return substr( $human_readable, 0, -2 );
+	}
+
+	/**
+	 * Return debug data in the format expected by Core's Site Health Info tab.
+	 *
+	 * @since 7.3.0
+	 *
+	 * @param array $debug {
+	 *     The debug information already compiled by Core.
+	 *
+	 *     @type string  $label        The title for this section of the debug output.
+	 *     @type string  $description  Optional. A description for your information section which may contain basic HTML
+	 *                                 markup: `em`, `strong` and `a` for linking to documentation or putting emphasis.
+	 *     @type boolean $show_count   Optional. If set to `true` the amount of fields will be included in the title for
+	 *                                 this section.
+	 *     @type boolean $private      Optional. If set to `true` the section and all associated fields will be excluded
+	 *                                 from the copy-paste text area.
+	 *     @type array   $fields {
+	 *         An associative array containing the data to be displayed.
+	 *
+	 *         @type string  $label    The label for this piece of information.
+	 *         @type string  $value    The output that is of interest for this field.
+	 *         @type boolean $private  Optional. If set to `true` the field will not be included in the copy-paste text area
+	 *                                 on top of the page, allowing you to show, for example, API keys here.
+	 *     }
+	 * }
+	 *
+	 * @return array $args Debug information in the same format as the initial argument.
+	 */
+	public static function core_debug_data( $debug ) {
+		$jetpack = array(
+			'jetpack' => array(
+				'label'       => __( 'Jetpack', 'jetpack' ),
+				'description' => __( 'Diagnostic information helpful to <a href="https://jetpack.com/contact-support">your Jetpack Happiness team</a>', 'jetpack' ),
```

Comment:
Should we make that link open a new tab?

Reply:
Not currently possible. Core runs this through wp_kses with only allowing `em`, `strong`, `a(href)`. See https://core.trac.wordpress.org/ticket/46878 for adding `target`.

That said, added in e60edd82e so it's already there. No harm adding it now.

I escaped the URL mainly because I'm thinking it makes sense to abstract that out to a constant? Or set one based on the local for sites where we have a translated support site (br?). In either case, just setting up a future PR.